src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.cs

```diff
@@ -16,7 +14,7 @@ namespace System.Collections.Immutable
     /// </summary>
     /// <typeparam name="T">The type of elements in the set.</typeparam>
     [DebuggerDisplay("Count = {Count}")]
-    [DebuggerTypeProxy(typeof(ImmutableListDebuggerProxy<>))]
+    [DebuggerTypeProxy(typeof(ImmutableEnumerableDebuggerProxy<>))]
```

Comment:
@AArnott This proxy is being used with all of the other immutable collections already, and those were validated to work by @stephentoub at https://github.com/dotnet/corefx/pull/16008#issuecomment-279889732. Plus I think @JonHanna has some corefx tests in place for the collection proxies. I think we can safely assume the proxy won't malfunction.

FWIW, I tried building this branch and copying the S.C.I dll into my app today, but I ran into a stubborn problem with binding redirects I've been stuck on for a couple of hours. I'm throwing in the towel a bit here.